### PR TITLE
Automatically port System.Text.Json.Serialization.JsonConverter`1

### PR DESCRIPTION
**Pull request by carlossanlop**
New batch.

### CODE REVIEW

**Comment by carlossanlop** [comment]
Looks good, thank you @rpetrusha. I'll set the right label.

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 1]
```diff
@@ -114,11 +122,19 @@
...
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="writer">The <see cref="T:System.Text.Json.Utf8JsonWriter" /> to write to.</param>
+        <param name="value">The value to convert.</param>
+        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> being used.</param>
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml]:
```suggestion
        <param name="options">The options being used.</param>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 2]
```diff
@@ -114,11 +122,19 @@
...
+        <summary>Write the value as JSON.</summary>
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml]:
```suggestion
        <summary>When overridden in a derived class, writes the specified value as JSON.</summary>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 3]
```diff
@@ -114,11 +122,19 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+A converter may throw any Exception, but should throw <cref>JsonException</cref> when the JSON cannot be created.
```
Comment (Jozkee) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml]:
```suggestion
A converter may throw any exception, but it should throw a <xref:System.Text.Json.JsonException> when the JSON cannot be created.
```

**Comment by Jozkee** [comment]
Whoops, a little bit late.

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 4]
```diff
@@ -59,9 +59,17 @@
       </Parameters>
       <Docs>
         <param name="typeToConvert">To be added.</param>
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Text.Json.Serialization/JsonConverter`1.xml]: I'm going to use what we have been using in other files:

```suggestion
        <param name="typeToConvert">The type of the converter.</param>
```

**Pull request merged**